tests/utils.spec.ts

```diff
@@ -66,14 +66,11 @@ describe('utils', function () {
 	describe('BalenaS3SourceBase', function () {
 		describe('isESRVersion', function () {
 			it('should return false for non-ESR versions following the original scheme', function () {
-				[
-					'2.7.8.dev',
-					'2.7.8.prod',
-					'2.80.3.0.dev',
-					'2.80.3.0.prod',
-				].forEach((v) => {
-					expect(BalenaS3SourceBase.isESRVersion(v)).to.be.false;
-				});
+				['2.7.8.dev', '2.7.8.prod', '2.80.3.0.dev', '2.80.3.0.prod'].forEach(
+					(v) => {
+						expect(BalenaS3SourceBase.isESRVersion(v)).to.be.false;
+					},
+				);
 			});
```

Comment:
nitpick: This is cosmetic - should maybe be a different PR.

Reply:
I've split it in another commit